vendor.conf

```diff
@@ -77,9 +77,10 @@ k8s.io/utils c2654d5206da6b7b6ace12841e8f359bb89b443c
 sigs.k8s.io/yaml v1.1.0
 
 # zfs dependencies
-github.com/containerd/zfs 9f6ef3b1fe5144bd91fe5855b4eba81bc0d17d03
-github.com/mistifyio/go-zfs 166add352731e515512690329794ee593f1aaff2
-github.com/pborman/uuid c65b2f87fee37d1c7854c9164a450713c28d50cd
+github.com/containerd/zfs 31af176f2ae84fe142ef2655bf7bb2aa618b3b1f
+github.com/mistifyio/go-zfs d5b163290a48f624cbf244ebe4e89ce38653064c
+github.com/pborman/uuid v1.2.0
+github.com/google/uuid v1.1.1
```

Comment:
We need both for now, as pborman/uuid is a thin wrapper around google/uuid; we can remove one if/once https://github.com/mistifyio/go-zfs/pull/72 is merged